plugins/generator-1.21.4/neoforge-1.21.4/templates/elementinits/items.java.ftl

```diff
@@ -122,6 +124,26 @@ public class ${JavaModName}Items {
 
 	<#if hasItemsWithProperties>

```

Comment:
Should be <#if hasItemsWithProperties || hasItemsWithLeftHandedProperty>

Maybe better name for hasItemsWithProperties would also be hasItemsWithCustomProperties as I suspect the original variable naming led to this confusion that led to this bug